cloudbuild.yaml

```diff
@@ -11,7 +11,7 @@ steps:
       - debug-image-push
       - importer-image-push
       - helm-chart-push
-      - kueue-viz-image-push
+      - kueue-viz-multiarch-push
```

Comment:
I am curious if k8s infra is okay with hosting these images for sig projects. 

We only push amd64 for kueue containers so I'm curious why we should do this for kueue-viz?

Reply:
Okay. I looked at this. Our kueue platform images do not match with what we are proposing for kueue-viz.

Reply:
PLATFORMS ?= linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
CLI_PLATFORMS ?= linux/amd64,linux/arm64,darwin/amd64,darwin/arm64
VIZ_PLATFORMS ?= linux/amd64,linux/arm64,linux/s390x,linux/ppc64le

I lean towards we should commit to supporting the same images for all platforms.